update.sh

```diff
@@ -189,7 +189,9 @@ print_lib32_support_arm64() {
 	RUN dpkg --add-architecture armhf && \
 	    apt-get update && \
 	    apt-get install --no-install-recommends -y \
-	    libc6:armhf
+	    fontconfig:armhf \ 
+	    libc6:armhf && \
+	    rm -rf /var/lib/apt/lists/*
```

Comment:
why do you need to delete the downloaded lists?

Reply:
Best practices for writing Dockerfiles:
When you clean up the apt cache by removing /var/lib/apt/lists reduces the image size, since the apt cache is not stored in a layer.
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Sorry for not responding sooner: Got it, makes sense and thanks for pointing to that, that was something I didn't know before :)

Reply:
Adding `rm -rf /var/lib/apt/lists/*` here is unnecessary since it is already added by [print_cleanup](https://github.com/FoxRomeo/openhab-docker/blob/0fbec0c74f7234d3d4180e4cd4853fc6ee2add1d/update.sh#L166).